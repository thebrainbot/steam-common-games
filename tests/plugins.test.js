jest.unmock('../src/plugins');
const plugins = require('../src/plugins');

describe('Plugins', () => {
  test('should have known loading phases', async () => {
    expect(Object.keys(plugins)).toMatchSnapshot();
  });
});
