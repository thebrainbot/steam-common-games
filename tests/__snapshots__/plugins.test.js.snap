// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plugins should have known loading phases 1`] = `
Array [
  "bootstrap",
  "main",
  "tail",
]
`;
